addons/mail/static/src/js/views/activity/activity_renderer.js

```diff
@@ -208,7 +208,8 @@ var ActivityRenderer = AbstractRenderer.extend({
                 var def = widget.appendTo($td).then(function () {
                     // replace clock by closest deadline
                     var $date = $('<div class="o_closest_deadline">');
-                    var date = new Date(activity_group.o_closest_deadline);
+                    var tmp = activity_group.o_closest_deadline.split('-');
+                    var date = new Date(`${tmp[1]}-${tmp[2]}-${tmp[0]}`);
```

Comment:
I'm not sure, but I think this will work also, and it's a little more clean.

```suggestion
                    var date = new Date(tmp[0], tmp[1] - 1, tmp[2]);
```

Reply:
it does work (tested on gmt-6 and our tz)